bolt-cli/src/cli.rs

```diff
@@ -131,6 +138,29 @@ pub struct SendCommand {
     pub devnet_sidecar_url: Option<Url>,
 }
 
+/// Command for registering validators in the Bolt network.
+#[derive(Debug, Clone, Parser)]
+pub struct RegisterCommand {
+    /// The chain for which the delegation message is intended.
+    #[clap(long, env = "CHAIN", default_value = "holesky")]
+    pub chain: Chain,
```

Comment:
Could we derive the chain from the RPC using eth_chainId? Cast uses the same pattern and it's a bit less verbose!